java-client-model/src/main/java/com/cumulocity/rest/representation/ErrorMessageRepresentation.java

```diff
@@ -0,0 +1,63 @@
+package com.cumulocity.rest.representation;
+
+import org.svenson.JSONProperty;
+
+public class ErrorMessageRepresentation extends AbstractExtensibleRepresentation {
+
+    // Application level error code
+    private String error;
+    // Short text description of the error
+    private String message;
+
+    private String info; //   1   URL to error description on the Internet.
```

Comment:
This comment looks differently than others in the class, and I do not understand `1`